### PR TITLE
Lpozo structure

### PR DESCRIPTION
**Pull request by lpozo**
Some more refactors and a new check.

### CODE REVIEW

**Comment by lpozo** [comment]
Hey @jima80525! Have you had time to check out this PR?

**Comment by jima80525** [comment]
> Hey @jima80525! Have you had time to check out this PR?



> Hey @jima80525! Have you had time to check out this PR?
Sorry I'm so slow on this one!  I'm on vacation visiting family.  Hopefully some time in the coming week.

**Comment by lpozo** [comment]
@jima80525 No problem! Enjoy your time with your family. I was just curious to confirm if you had seen the PR.

File: README.md [hunk 1]
```diff
@@ -1,82 +1,93 @@
...
 ## Helping Out
 
-I'm not sure if this will be useful to others, so until I see signs of life, I'm not going to bother with a full code of conduct and rules/guidelines for adding to the project.  Right now the rules are basically: 
+I'm not sure if this will be useful to others, so until I see signs of life, I'm not going to bother with a full code of conduct and rules/guidelines for adding to the project. Right now the rules are basically:
 
 * Be Nice
 * Contact me (jima) if you've got questions/comments/suggestions.
 
-This project is in very early stages and I'm not certain it will continue as a stand alone project. Watch this space!
+This project is in very early stages and I'm not certain it'll continue as a stand alone project. Watch this space!
```
Comment (jima80525) [inline README.md:72]: OK.  This change cracked me up!  Nice

File: rplint/__main__.py [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+        if name.endswith("Check")
+    }
+    checks["LineLengthCheck"].line_length = line_length
+    lines = input_file.readlines()
+    for check in checks.values():
+        check.run(lines)
+        if check:
+            click.secho(check, fg="red")
+        else:
+            click.secho(f"{check}... Passes!", fg="green")
```
Comment (jima80525) [inline rplint/__main__.py:39]: That's nice.  I like it

File: rplint/checks.py [hunk 3]
```diff
@@ -285,35 +284,17 @@ def check_line(self, lineno, line):
...
+    def __init__(self):
+        super().__init__()
+        self.title = "Spaces in Line Check"
+
+    def check_line(self, lineno, line):
+        if line.endswith(" \n"):
+            self.register_error(
+                lineno, self.error_format % "trailing whitespace"
+            )
+        if not self.in_code_block and "|" not in line and "  " in line:
```
Comment (jima80525) [inline rplint/checks.py:297]: I'm terrible at this one.  I learned to type on a manual typewriter in the days when it was required to put two spaces after periods.  LOTS of muscle memory.  Thanks for adding this!

**Review by jima80525 — APPROVED** [review]
awesome!

**Pull request merged**